src/utilities/utilities.jl

```diff
@@ -616,7 +655,7 @@ it out automatically.
 `root` is the the directory where `git` gets run. `varname` is just informational and used
 to construct the warning messages.
 """
-function git_remote_head_branch(varname, root; remotename = "origin", fallback = "master")
+function git_remote_head_branch(varname, root; fallback = "master")
```

Comment:
The docstring still references `remotename`. So either that needs to be adjusted. Alternatively, keep the kwarg here, but with a new default value `nothing`; and then later only call `getremotename` if `remotename === nothing` ?